app/models/provider.rb

```diff
@@ -26,6 +26,7 @@ class Provider < ApplicationRecord
 
   has_many :recommendations_uploads, class_name: "BulkUpdate::RecommendationsUpload"
   has_many :recommendations_upload_rows, class_name: "BulkUpdate::RecommendationsUploadRow", through: :recommendations_uploads
+  has_one :latest_recommendations_upload, -> { order(created_at: :desc) }, class_name: "BulkUpdate::RecommendationsUpload"
```

Comment:
Is this being used anywhere? Not sure what the use case for this will be as there's no 'return to latest' or 'pick up where you left off'

Reply:
You're correct. I removed the usage, but forgot to remove the scope. Thanks for the spot